ci/prow/create_boskos_projects.sh

```diff
@@ -36,10 +36,10 @@ LAST_INDEX=$(grep "${BOSKOS_PROJECT_PREFIX}" ${BOSKOS_RESOURCE_FILE} | grep -o "
 [[ -z "${LAST_INDEX}" ]] && LAST_INDEX=0
 for (( i=1; i<=${NUMBER}; i++ )); do
   PROJECT="$(printf ${BOSKOS_PROJECT_PREFIX}%02d $(( ${LAST_INDEX} + i )))"
-  # This Folder ID is google.com/google-default
+  # This Folder ID is google.com/knative
   # If this needs to be changed for any reason, GCP project settings must be updated.
   # Details are available in Google's internal issue 137963841.
```

Comment:
Can you also update the issue number? Looks it's outdated.

Reply:
No, it's not. Please see the issue referenced above.

Reply:
I see, I misunderstood it.

/lgtm